csfieldguide/templates/interactives/confusing-error.html

```diff
@@ -0,0 +1,23 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+	<div class="container">
+		<div class="row justify-content-center">
+		  <div class="card col-lg-8 col-xs-12">
+		  	<div class="card-body text-left">
+		  		<h6 class="card-title">{% trans "This configuration will cause your backup device's status light to blink amber. Are you sure you want to continue?" %}</h6>
+		  		<img class="float-left icon-4 mr-2" src="{% static 'interactives/confusing-error/img/icons8-data-backup-100.png' %}">
```

Comment:
Because we are using an icons8 image, we need to add attribution in the footer. Could you please check what is required and add it? I have done it here: https://github.com/uccser/ktam-content-assistant/blob/develop/templates/base.html#L136 but best to check that's correct.